README.md

```diff
@@ -1,15 +1,42 @@
-# neuroid-reactnative-sdk
+# NeuroID React-Native Mobile SDK
 
-## Installation
+![Neuro-ID Logo](/docs/images/NeuroID_Logo.png)
 
-Visit https://neuro-id.readme.io/docs/react-native-installation for full installation documentation
+## Overview
 
-## Contributing
+NeuroID's React-Native Mobile SDK makes it simple to embed behavioral analytics inside your mobile application. With a few lines of code, you can connect your app with the NeuroID platform and make informed decisions about your users.
```

Comment:
React Native should not be hyphenated. Yes this is the most pedantic review, sorry LOL :( Lots of instances below, I will do you the grace of not commenting on each one...